### ISSUE TITLE
ncm-ntp problem

### ISSUE THREAD

**Issue by quattor-releaser** [quattor/release#140]
Hi,
currenlty ntp component configures `/etc/ntp.conf` and then starts `/etc/init.d/ntpd`. When a node is out of sync, simply doing this is not enough, and the node stays out of sync. It should be good to perform a `ntpdate <time-server>` before starting the daemon. 

Reported by: chierici

**Comment by ned21** [comment]
This is an issue with ntpd initscript not quattor so marking won't fix. (It may well be better handled in newer versions of EL too.)

**Comment by stdweird** [comment]
fyi, el7 has both ntpd and ntpdate units to handle this. if you enable both, the dependencies between them will "do the right thing"
